### PR TITLE
space missing between the two function

### PR DESCRIPTION
**Pull request by saurabhiiit**
Space Missing

### CODE REVIEW

**Comment by saurabhiiit** [comment]
Sure, but for the better code practices the newline should be present

File: babel/localtime/_unix.py [hunk 1]
```diff
@@ -26,6 +26,7 @@ def _tz_from_env(tzenv):
         raise pytz.UnknownTimeZoneError(
             "tzlocal() does not support non-zoneinfo timezones like %s. \n"
             "Please use a timezone in the form of Continent/City")
+
```
Comment (sils1297) [inline babel/localtime/_unix.py]: you add trailing whitespace here!

**Comment by saurabhiiit** [comment]
@sils1297 yes because of better code practices,  we need to add whitespace between the function, except this , everywhere in the code was having two whitespace between function

**Comment by sils1297** [comment]
you add trailing whitespace which is not ok, this is not about newlines!

**Pull request closed without merging**